src/reanimated2/js-reanimated/JSReanimated.ts

```diff
@@ -50,6 +50,22 @@ export default class JSReanimated {
     );
   }
 
+  createBackgroundQueue(_name: string): BackgroundQueue {
+    throw new Error(
+      '[Reanimated] createBackgroundQueue is not available in JSReanimated.'
+    );
+  }
+
+  scheduleOnBackgroundQueue<T>(
+    _backgroundQueue: BackgroundQueue,
+    _workletRuntime: WorkletRuntime,
+    _worklet: ShareableRef<T>
+  ): BackgroundQueue {
```

Comment:
WDYT? Such a change could be made with all function throwing error in this file.
```suggestion
  ): never {
```

Reply:
Thanks for this suggestion, indeed `never` works. I think we could go one step ahead and also remove the unused arguments along with the template and also remove the return type. I remember I tested it some time ago but it didn't work for me, but it's okay now. I'd be more than happy to change all methods in the next PR.
```ts
  scheduleOnBackgroundQueue() {
    throw new Error(
      '[Reanimated] scheduleOnBackgroundQueue is not available in JSReanimated.'
    );
  }
```